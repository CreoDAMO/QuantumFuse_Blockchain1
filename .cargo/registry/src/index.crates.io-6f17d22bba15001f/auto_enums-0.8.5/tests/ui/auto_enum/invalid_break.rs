// SPDX-License-Identifier: Apache-2.0 OR MIT

fn closure() -> impl Fn() {
    || break
}

fn async_block() -> impl std::future::Future<Output = ()> {
    async { break }
}

fn main() {}
