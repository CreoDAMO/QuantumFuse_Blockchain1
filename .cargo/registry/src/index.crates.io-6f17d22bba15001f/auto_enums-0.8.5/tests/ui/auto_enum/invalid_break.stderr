error[E0267]: `break` inside of a closure
 --> tests/ui/auto_enum/invalid_break.rs:4:8
  |
4 |     || break
  |     -- ^^^^^ cannot `break` inside of a closure
  |     |
  |     enclosing closure

error[E0267]: `break` inside of an `async` block
 --> tests/ui/auto_enum/invalid_break.rs:8:13
  |
8 |     async { break }
  |     --------^^^^^--
  |     |       |
  |     |       cannot `break` inside of an `async` block
  |     enclosing `async` block
